locales/languages/en.json

```
@@ -2324,5 +2324,67 @@
+    "bluetooth_off": "Bluetooth is off",
+    "bluetooth_off_message": "Please turn on Bluetooth for your device",
+    "bluetooth_access_blocked": "Ledger needs Bluetooth access to pair with your mobile device.",
+    "bluetooth_access_blocked_message": "If you want to pair your Ledger device using Bluetooth, enable it in Settings & try again.",
+    "location_access_blocked": "Ledger needs location access to pair with your mobile device.",
+    "location_access_blocked_message": "If you want to pair your Ledger device using Bluetooth, you’ll need to enable location access in Settings & try again.",
+    "bluetooth_scanning_error": "Error while scanning the device",
+    "bluetooth_scanning_error_message": "Please make sure your device is unlocked and the Ethereum app is running",
+    "bluetooth_connection_failed": "Bluetooth connection failed",
+    "bluetooth_connection_failed_message": "Please make sure your Ledger is unlocked and your Bluetooth is enabled",
+    "ethereum_app_open": "Ledger requires confirmation",
+    "ethereum_app_open_message": "Please confirm on your device to open Ethereum app. Press OK once done.",
+    "ethereum_app_unconfirmed_error": "You have denied the request to open the Ethereum app.",
+    "failed_to_open_eth_app": "Failed to open the Ethereum app.",
+    "ethereum_app_open_error": "Please install/accept Ethereum app on your Ledger device.",
+    "running_app_close": "Failed to close the running app.",
+    "running_app_close_error": "Failed to close the running app on your Ledger device.",
+    "ethereum_app_not_installed": "Ethereum app not installed.",
+    "ethereum_app_not_installed_error": "Please install the Ethereum app on your Ledger device.",
+    "ledger_is_locked": "Ledger is locked",
+    "unlock_ledger_message": "Please unlock your Ledger device",
+    "cannot_get_account": "Cannot get account",
+    "connect_ledger": "Connect Ledger",
+    "looking_for_device": "Looking for device",
+    "ledger_reminder_message": "Please make sure your Ledger Nano X is:",
+    "ledger_reminder_message_step_one": "1. Unlocked",
+    "ledger_reminder_message_step_two": "2. The Ethereum app is installed",
+    "ledger_reminder_message_step_three": "3. Bluetooth is enabled",
+    "ledger_reminder_message_step_four": "4. Location is enabled",
+    "available_devices": "Available devices",
+    "retry": "Retry",
+    "continue": "Continue",
+    "confirm_transaction_on_ledger": "Confirm transaction by pressing two buttons on your Ledger",
+    "bluetooth_enabled_message": "Make sure Bluetooth is enabled",
+    "device_unlocked_message": "Device is unlocked",
+    "blind_signing_message": "Blind signing is enabled",
+    "ledger_disconnected": "Your device got disconnected",
+    "ledger_disconnected_error": "The connection to your device has been lost. Please try again.",
+    "unknown_error": "Unexpected error occurred.",
+    "unknown_error_message": "An unexpected error occured. Please try again.",
```

Error are most of the time unexpected...
```suggestion
    "unknown_error_message": "An unknown error occured. Please try again.",
```
But I would even fusion this with the error_occured one... knowing it's unexpected or unknown doesn't add anything ot the user understanding. It just makes them more anxious...